packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.8+15
+
+* Fixes typo "intialize" in doc comment.
```

Comment:
Since the comments are only in tests, you should be able to revert the CHANGELOG and pubspec changes in both packages.

Reply:
Indeed, thanks for pointing this out!